### ISSUE TITLE
feat: RTL support

### ISSUE THREAD

**Issue by liorkesos** [open-webui/open-webui#2134]
**Is your feature request related to a problem? Please describe.**
To enable multi-lingual support for Arabic , hebrew , Farsi etc most apps have RTL capabilities.
This is needed for optimal experience for these languages

**Describe the solution you'd like**
The CSS implementation needs to know that certain languages should have a RTL layout

**Comment by justinh-rahb** [comment]
Great idea, a PR would be welcomed!

**Comment by liorkesos** [comment]
I'll issue a PR for the hebrew translation and see how I can bring them both to a stage where they give a good hebrew user experience.

**Comment by LebToki** [comment]
Here's a possible solution:

First, you need to detect the chosen language and store it in a variable. 
You can use the navigator.language property or a library like i18n-js to detect the language.

Once you have the chosen language, you can use JavaScript to add the dir attribute to the HTML tag. 
You can use the setAttribute method to add the attribute.
To use the Tailwind classes, you need to add the rtl or ltr class to the HTML tag based on the chosen language. You can use the classList.add method to add the class.

Here's some code to get you started:

// Detect the chosen language
const language = navigator.language || navigator.userLanguage;

// Add the dir attribute to the HTML tag
document.documentElement.setAttribute('dir', language === 'ar' || language === 'he' ? 'rtl' : 'ltr');

// Add the Tailwind classes
document.documentElement.classList.add(language === 'ar' || language === 'he' ? 'rtl' : 'ltr');

This is also try that then we can pass font-family and css property to get things shape up

**Comment by idomamia** [comment]
Hey, I've recently started developing something similar.
In the settings, under the "Interface" section, I implemented a toggle button that allows users to switch between LTR and RTL layouts.
Additionally, I thought it would be useful to restrict the RTL layout to only apply to the Chat section (Code blocks will always remain LTR). What are your thoughts on this?

Here's a gif of my implementation:
![Open WebUI (1) - Copy](https://github.com/open-webui/open-webui/assets/48285277/a3086a83-0e22-4e1b-ba19-10d5655710ab)
